tests/git/utils.py

```diff
@@ -0,0 +1,38 @@
+import importlib
+import sys
+import tempfile
+from pathlib import Path
+from textwrap import dedent
+
+import pytest
+from snowflake.cli.api.config import config_init
+
+
+@pytest.fixture(autouse=True)
+def enable_snowgit_fixture(snowflake_home):
+    _reload_config_to_enable_snowgit(snowflake_home / "config.toml")
+    yield
+
+
+def reload_config_to_enable_snowgit():
```

Comment:
can't we run snowcli with `--config-file` option (`runner.invoke_with_config`) instead of modifying the default config?

Reply:
fixed